samples/sensor/fxos8700-hid/sample.yaml

```diff
@@ -1,7 +1,7 @@
 sample:
   name: USB HID accelerometer mouse sample
 tests:
-  sample.usb.hid-mouse:
+  sample.usb.fxos8700-hid:
```

Comment:
Looks OK, though it can be also `sample.sensor.usb.fxos8700-hid` since this is inside  `samples/sensor/`